Client/GameWorld/Repositories/ResourceRepository.cs

```diff
@@ -1,17 +1,32 @@
-﻿using Microsoft.Data.SqlClient;
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
 using GameWorld.Resources.Utils;
 using GameWorld.Models;
+using Newtonsoft.Json;
 
 namespace GameWorld.Repositories
 {
     public class ResourceRepository : IResourceRepository
     {
-        private readonly string connectionString = DatabaseHelper.GetDatabaseFilePath();
+        // private readonly string connectionString = DatabaseHelper.GetDatabaseFilePath();
+        private readonly HttpClient httpClient;
+        private readonly string baseURL;
+
+        public ResourceRepository(HttpClient httpClient, string baseURL)
+        {
+            this.httpClient = httpClient;
+            this.baseURL = baseURL;
+        }
```

Comment:
move baseUrl to Resources/Utils/Apis
we should have all api paths in one place, there is no need to inject the url.